.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -33,15 +33,20 @@ body:
   - type: dropdown
     attributes:
       label: NuGet package version
-      description: If you are seeing your issue on older versions please try the latest
+      description: If you are seeing your issue on older versions please try the latest stable release
```

Comment:
This says "latest _stable_ release", but then the bullets include non-stable (preview) releases.